pypolestar/proto/polestar_battery.proto

```
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package pccs.vehiclestates.entities.battery.v1;
 
-import "common.proto";
+import "polestar_common.proto";
```

Buf (1.69.0)

[error] imported file does not exist

(COMPILE)